download_mac.sh

```diff
@@ -2,4 +2,10 @@
 
 sudo apt-get install -qq sshpass
 gem install package_cloud
+
+# Download the mac library.
 sshpass -p $SSH_PASS sftp -oStrictHostKeyChecking=no -oPort=2223 -r $SSH_USER@185.19.29.18:/upload down/
+# Cleanup.
+sshpass -p $SSH_PASS sftp -oStrictHostKeyChecking=no -oPort=2223 $SSH_USER@185.19.29.18:/upload <<< $'rm 'libtraildbjava-$TRAVIS_TAG'.dylib'
```

Comment:
Yes it works but maybe it can be improved. In the end it just concatenate 'rm ' witth 'libtraildbjava-$TRAVIS_TAG' with '.dylib' but it looks like the $ is maybe useless. I'll check.